redash/query_runner/graphite.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, configuration):
             self.auth = None
 
         self.verify = self.configuration.get("verify", True)
-        self.base_url = "%s/render?format=json&" % self.configuration['url']
+        self.base_url = "%s/render?format=json&target=" % self.configuration['url']
```

Comment:
After some more thought -- it's better to maintain backward compatibility here. I guess you can check if the query starts with `target=` and then do nothing and if it isn't, add it. Then we can safely merge this one.